src/vehicle.cpp

```diff
@@ -5624,7 +5551,7 @@ bool vehicle::automatic_fire_turret( int p, const itype &gun, const itype &ammo,
     // Drain a ton of power
     tmp_ups.charges = drain( fuel_type_battery, 1000 );
     tmp.worn.insert( tmp.worn.end(), tmp_ups );
-    tmp.fire_gun( xtarg, ytarg, burst );
+    tmp.fire_gun( targ, abs( parts[p].mode ) );
```

Comment:
Is the behavior change intended? The original code had `abs(mode)>1`, this here is `abs(mode)>0`.